src/utils.ts

```diff
@@ -88,3 +56,20 @@ export function toggleLogs() {
     workspace.nvim.command(Commands.OPEN_LOGS);
   }
 }
+
+export function detectLaunchConfigurationChanges(): void {
+  metalsLanguageClient.detectLaunchConfigurationChanges(
+    workspace,
+    ({ message, reloadWindowChoice, dismissChoice }) => {
+      // NOTE: I used to just have a prompt here, but was having issues
```

Comment:
Random, but you're not using a pre-8 version of Vim are you?

Reply:
this is your note 😅

Reply:
Ugh, you're right. I even looked this up because I thought "was this me? I did a grep `NOTE`, and didn't see this in my repo..... _looks again and realizes it was `Note`_.

Well anyways, I had problems with that it seem 🤣